flowapi/flowapi/permissions.py

```diff
@@ -210,7 +210,9 @@ async def query_to_scopes(query_dict):
     except Exception:
         raise BadQueryError
     agg_unit = await get_agg_unit(query_dict)
-    return [f"{agg_unit}:{tl_query_name}:{query_name}" for query_name in query_list]
+    return list(
```

Comment:
Do we need this to be a list, or could we return a set instead? (I don't think it matters much either way, but if we return a set it's perhaps clearer that the scopes will be non-duplicated)

Reply:
It's compared to other lists when we check it against the requested scopes; now I say that, a set would be better there too